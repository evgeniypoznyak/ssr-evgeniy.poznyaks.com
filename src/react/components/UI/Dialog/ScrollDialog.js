import React from 'react';
import Button from '@material-ui/core/Button';
import Dialog from '@material-ui/core/Dialog';
import DialogActions from '@material-ui/core/DialogActions';
import DialogContent from '@material-ui/core/DialogContent';
import DialogTitle from '@material-ui/core/DialogTitle';
import {makeStyles} from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
    title: {
        flexGrow: 1,
    },
    content: {
        padding: 10,
    },
}));

export default function ScrollDialog(props) {
    const classes = useStyles();
    const [open, setOpen] = React.useState(false);
    const [scroll, setScroll] = React.useState('paper');

    const handleClickOpen = scrollType => () => {
        setOpen(true);
        setScroll(scrollType);
    };

    function handleClose() {
        setOpen(false);
    }

    return (
        <div>
            <Button onClick={handleClickOpen('paper')}>Read more</Button>
            <Dialog
                open={open}
                onClose={handleClose}
                scroll={scroll}
                aria-labelledby="scroll-dialog-title"
                fullWidth={true}
                maxWidth={'lg'}
            >
                <DialogTitle id="scroll-dialog-title">
                    {props.title}
                </DialogTitle>
                <DialogContent dividers={true} className={classes.content}>
                    <DialogContent className={classes.content}>
                        {props.children}
                    </DialogContent>
                </DialogContent>
                <DialogActions>
                    <Button onClick={handleClose} color="primary">
                        Close
                    </Button>
                </DialogActions>
            </Dialog>
        </div>
    );
}
