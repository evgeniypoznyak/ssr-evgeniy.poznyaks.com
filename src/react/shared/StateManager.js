import {createContext} from 'react';

export const sortSkills = skills => {
    const sortedSkills = skills.slice().sort(compareByOrder);
    return sortedSkills.reduce((acc, curr) => {
        if (curr['panel'] === 'left') {
            acc['left'].push(curr);
        }
        if (curr['panel'] === 'right') {
            acc['right'].push(curr);
        }
        return acc;
    }, {left: [], right: []});
};

export const compareByOrder = (a, b) => {
    if (a.order < b.order) {
        return -1;
    }
    if (a.order > b.order) {
        return 1;
    }
    return 0;
};

export const State = createContext(null);
