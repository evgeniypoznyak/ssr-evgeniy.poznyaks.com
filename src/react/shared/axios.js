import axios from 'axios';


axios.interceptors.request.use(
    config => {
        let token = '';
        // eslint-disable-next-line no-undef
        if (__isBrowser__) {
            if (localStorage) {
                token = localStorage.getItem('token');
            }
        }

        config.headers['x-auth-token'] = token;
        return config;
    },
    error => Promise.reject(error),
);

export default axios;
