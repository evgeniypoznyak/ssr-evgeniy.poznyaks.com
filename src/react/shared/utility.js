export const updateObject = (oldObject, updatedProperties) => {
    return {
        ...oldObject,
        ...updatedProperties,
    };
};

export const checkValidity = (value, rules) => {
    let isValid = true;
    if (!rules) {
        return true;
    }
    if (typeof (value) == 'undefined' || value == null) {
        return false;
    }

    if (rules.required) {
        isValid = ('' + value).trim() !== '' && isValid;
    }

    if (rules.minLength) {
        isValid = ('' + value).trim().length >= rules.minLength && isValid;
    }

    if (rules.maxLength) {
        const lenght = ('' + value).trim().length;
        isValid = lenght <= rules.maxLength && lenght > 0 && value !== '' && isValid;
    }

    if (rules.isEmail) {
        // eslint-disable-next-line max-len
        const pattern = /[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/;
        isValid = pattern.test(value) && isValid;
    }

    if (rules.isNumeric) {
        const pattern = /^\d+$/;
        isValid = pattern.test(value) && isValid;
    }

    return isValid;
};

export const browserLanguage = function browserLocale() {
    if (navigator.languages && navigator.languages.length) {
        return navigator.languages[0];
    } else if (navigator.userLanguage) {
        return navigator.userLanguage;
    }
    return navigator.language;
};

export const stripHtmlTags = element => element.replace(/<\/?[^>]+(>|$)/g, '');
