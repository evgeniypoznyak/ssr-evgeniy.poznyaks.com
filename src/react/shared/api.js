import axios from './axios';
import {skills} from './initialData/skills';

const url = process.env.API_GATEWAY ||
    'https://api.evgeniy.poznyaks.com'
    // 'http://localhost:2222'
;
const apiPublic = process.env.API_GATEWAY_PUBLIC ||
    'https://api.evgeniy.poznyaks.com'
    // 'http://localhost:2222'
;

const selfUrl = process.env.SELF_URL ||
    ''
    // 'http://localhost:3001'
;

export async function fetchSkills(path = 'all') {
    const biography = await getBiography();
    try {
        console.log('[GET] processing url: ', `${url}/api/skills/`);
        const result = await axios.get(`${url}/api/skills/`);
        return {
            skills: result.data.skills,
            biography,
        };
    } catch (e) {
        console.log('Error: ', e.message);
        console.log('Something wrong happened with API Gateway! Returning default response.');
        console.log('returning regular skills...');
        return {skills, biography};
    }
}

export async function sendEmail(data) {
    try {
        const url = `${apiPublic}/api/email`;
        console.log('[POST] processing url: ', `${url}`);
        const result = await axios.post(url, data);
        return result.data;
    } catch (e) {
        const error = {
            status: e.response.data.status,
            message: e.response.data.message,
            errorMessage: e.message,
        };
        console.log('Error: ', e.message);
        console.log('Something wrong happened with API Gateway! Returning default response.');
        return error;
    }
}

export async function fetchBiography(path) {
    console.log(path);
    const url = '/biography';
    try {
        console.log('[GET] processing url: ', `${url}/`);
        const result = await axios.get(`${url}/`);
        return result.data;
    } catch (e) {
        console.log('Error: ', e.message);
        console.log('Something wrong happened with API Gateway! Returning default response.');
        console.log('returning regular skills...');
        return {skills};
    }
}

export async function signInIntoApiGateway(data) {
    try {
        console.log('[POST] processing url: ', `${apiPublic}/api/auth`);
        const result = await axios.post(`${apiPublic}/api/auth`, {...data});
        return result.data;
    } catch (e) {
        console.log('Error: ', e.message);
        console.log('Something wrong happened with API Gateway! Returning default response.');
        return {token: ''};
    }
}

export async function verifyToken() {
    try {
        // eslint-disable-next-line no-undef
        if (__isBrowser__ && localStorage.getItem('token') && localStorage.getItem('token').length > 0) {
            return 'OK';
        }
    } catch (e) {
        console.log('Error: ', e.message);
        console.log('Something wrong happened with API Gateway! Returning default response.');
        return {token: ''};
    }
}

export async function verifyTokenApi() {
    try {
        const url = `${apiPublic}/api/auth/verify`;
        console.log('[POST] processing url: ', `${url}`);
        const result = await axios.post(url);
        return result.data;
    } catch (e) {
        console.log('Error: ', e.message);
        console.log('Something wrong happened with API Gateway! Returning default response.');
        return {token: ''};
    }
}

export async function patchSkill(skill) {
    try {
        const url = `${apiPublic}/api/skills`;
        console.log('[PATCH] processing url: ', `${url}`);
        const result = await axios.patch(url, {skill});
        return result.data;
    } catch (e) {
        console.log('Error: ', e.message);
        console.log('Something wrong happened with API Gateway! Returning default response.');
        return 'Skill not been patched';
    }
}

export async function deleteSkill(skillId) {
    try {
        const url = `${apiPublic}/api/skills/` + skillId;
        console.log('[DELETE] processing url: ', `${url}`);
        const result = await axios.delete(url);
        return result.data;
    } catch (e) {
        console.log('Error: ', e.message);
        console.log('Something wrong happened with API Gateway! Returning default response.');
        return 'Skill not been deleted';
    }
}

export async function uploadResume(resume) {
    try {
        const verification = await verifyTokenApi();
        if (verification === 'OK') {
            const url = '/resume';
            console.log('[POST] processing url: ', `${url}`);
            return await axios.post(url, resume);
        }
    } catch (e) {
        console.log('Error: ', e.message);
        console.log('Something wrong happened with this server! Returning default response.');
        return 'Resume not been uploaded';
    }
}


export async function putSkills(skills) {
    try {
        const verification = await verifyTokenApi();
        if (verification === 'OK') {
            const url = `${apiPublic}/api/skills`;
            console.log('[PATCH] processing url: ', `${url}`);
            const result = await axios.put(url, {skills});
            return result.data;
        }
    } catch (e) {
        console.log('Error: ', e.message);
        console.log('Something wrong happened with API Gateway! Returning default response.');
        return 'Skill not been patched';
    }
}


export async function uploadBiography(biography) {
    try {
        const verification = await verifyTokenApi();
        if (verification === 'OK') {
            const url = '/biography';
            console.log('[POST] processing url: ', `${url}`);
            return await axios.post(url, {biography});
        }
    } catch (e) {
        console.log('Error: ', e.message);
        console.log('Something wrong happened with this server! Returning default response.');
        return 'Resume not been uploaded';
    }
}

export async function getBiography() {
    try {
        const url = `${selfUrl}/biography`;
        console.log('[GET] processing url: ', `${url}`);
        const result = await axios.get(url);
        return result.data.biography;
    } catch (e) {
        console.log('Error: ', e.message);
        console.log('Something wrong happened with this server! Returning default response.');
        return 'Biography not been uploaded';
    }
}


export const getSkillsByPanes = skills => {
    const sortedSkills = skills.slice().sort(sortByOrder);
    return sortedSkills.reduce((acc, curr) => {
        if (curr['panel'] === 'left') {
            acc['left'].push(curr);
        }
        if (curr['panel'] === 'right') {
            acc['right'].push(curr);
        }
        return acc;
    }, {left: [], right: []});
};

export const sortByOrder = (a, b) => {
    if (a.order * 1 < b.order * 1) {
        return -1;
    }
    if (a.order * 1 > b.order * 1) {
        return 1;
    }
    return 0;
};
