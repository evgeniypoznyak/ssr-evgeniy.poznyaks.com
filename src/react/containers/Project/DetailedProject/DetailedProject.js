import React, {Fragment} from 'react';
import {makeStyles} from '@material-ui/core/styles';
import Paper from '@material-ui/core/Paper';
import {getRandomString} from '../../../shared/utility';
import {Grid} from '@material-ui/core';


const useStyles = makeStyles(theme => ({
    root: {
        padding: theme.spacing(3, 2),
    },
    paper: {
        'padding': theme.spacing(3, 2),
        'fontSize': '2rem',
        '@media (max-width: 960px)': {
            fontSize: '1.3rem',
        },
        '@media (max-width: 600px)': {
            fontSize: '1rem',
        },
    },
    imageWrapper: {
        width: '100%',
        textAlign: 'center',
    },
    image: {
        maxWidth: '100%',
    },
    descriptionElement: {
        marginBottom: 20,
        padding: 10,
    },
    divider: {
        marginTop: 20,
        marginBottom: 20,
    },
}));

export default function DetailedProject(props) {
    const classes = useStyles();

    return (
        <Fragment>
            {props.description.map(d => {
                return (
                    <Fragment key={getRandomString(99)}>
                        <Paper className={classes.descriptionElement}>
                            <Grid
                                container
                                direction="row"
                                justify="center"
                                spacing={1}
                                alignItems="center">
                                <Grid item xs={12} md={6}>
                                    {d.picture ? <Fragment>
                                        <div className={classes.imageWrapper}>
                                            <img alt="detailed" src={d.picture} className={classes.image}/>
                                        </div>
                                    </Fragment> : null}
                                </Grid>
                                <Grid item xs={12} md={6}>
                                    <Paper className={classes.paper}>{d.description}</Paper>
                                </Grid>
                            </Grid>
                        </Paper>
                    </Fragment>
                );
            })}
        </Fragment>
    );
}
