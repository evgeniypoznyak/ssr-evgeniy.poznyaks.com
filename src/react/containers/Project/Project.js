import React from 'react';
import {makeStyles, useTheme} from '@material-ui/core/styles';
import Card from '@material-ui/core/Card';
import CardHeader from '@material-ui/core/CardHeader';
import CardMedia from '@material-ui/core/CardMedia';
import CardContent from '@material-ui/core/CardContent';
import CardActions from '@material-ui/core/CardActions';
import IconButton from '@material-ui/core/IconButton';
import Typography from '@material-ui/core/Typography';
import Avatar from '@material-ui/core/Avatar';
import Grid from '@material-ui/core/Grid';
import MobileStepper from '@material-ui/core/MobileStepper';
import Paper from '@material-ui/core/Paper';
import Button from '@material-ui/core/Button';
import PublicIcon from '@material-ui/icons/Public';
import KeyboardArrowLeft from '@material-ui/icons/KeyboardArrowLeft';
import KeyboardArrowRight from '@material-ui/icons/KeyboardArrowRight';
import Box from '@material-ui/core/Box';
import ScrollDialog from '../../components/UI/Dialog/ScrollDialog';
import DetailedProject from './DetailedProject/DetailedProject';
import {getMonthAndDay} from '../../shared/utility';

const useStyles = makeStyles(theme => ({
    sliderRoot: {
        maxWidth: 400,
        flexGrow: 1,
    },
    card: {
        maxWidth: 345,
        margin: 10,
        marginTop: 30,
        boxShadow: '10px 10px 31px -5px rgba(0,0,0,0.55)',
    },
    media: {
        height: 0,
        paddingTop: '56.25%', // 16:9
    },
    expand: {
        transform: 'rotate(0deg)',
        marginLeft: 'auto',
        transition: theme.transitions.create('transform', {
            duration: theme.transitions.duration.shortest,
        }),
    },
    expandOpen: {
        transform: 'rotate(180deg)',
    },
    avatar: {},
    smallAvatar: {
        width: 25,
        height: 25,
    },
    header: {
        display: 'flex',
        alignItems: 'center',
        height: 50,
        paddingLeft: theme.spacing(4),
        backgroundColor: theme.palette.background.default,
    },
    img: {
        height: 'auto',
        maxWidth: 400,
        overflow: 'hidden',
        display: 'block',
        width: '100%',
    },

}));


export default function Project(props) {
    const images = props.project.images;
    const classes = useStyles();
    const theme = useTheme();
    const [activeStep, setActiveStep] = React.useState(0);
    const maxSteps = images.length;

    function handleNext() {
        setActiveStep(prevActiveStep => prevActiveStep + 1);
    }

    function handleBack() {
        setActiveStep(prevActiveStep => prevActiveStep - 1);
    }

    let slider = null;

    if (
        props.project.images &&
        props.project.images.length > 0 &&
        props.project.images[0].path &&
        props.project.images[0].path.length > 0
    ) {
        slider = <div className={classes.sliderRoot}>

            <Paper square elevation={0} className={classes.header}>
                <Typography variant="subtitle1">{images[activeStep].label}</Typography>
            </Paper>
            <img
                className={classes.img}
                src={images[activeStep].path}
                alt={images[activeStep].label}
            />
            <MobileStepper
                steps={maxSteps}
                position="static"
                variant="text"
                activeStep={activeStep}
                nextButton={
                    <Button size="small" onClick={handleNext} disabled={activeStep === maxSteps - 1}>
                        Next
                        {theme.direction === 'rtl' ? <KeyboardArrowLeft/> : <KeyboardArrowRight/>}
                    </Button>
                }
                backButton={
                    <Button size="small" onClick={handleBack} disabled={activeStep === 0}>
                        {theme.direction === 'rtl' ? <KeyboardArrowRight/> : <KeyboardArrowLeft/>}
                        Back
                    </Button>
                }
            />
        </div>;
    }
    let cardImage = null;
    if (props.project.cardCover.length > 0) {
        cardImage = <CardMedia
            className={classes.media}
            image={props.project.cardCover}
            title={props.project.name}
        />;
    }

    let avatarProject = null;
    if (props.project.logo && props.project.logo.length > 0) {
        avatarProject = <Avatar aria-label="Project" src={props.project.logo} className={classes.avatar}/>;
    }

    let dateCreated = null;
    if (props.project.dateCreated && props.project.dateCreated.length > 0) {
        dateCreated = 'Created: ' + getMonthAndDay(props.project.dateCreated);
    }

    let gitHubProjectLink = null;
    if (props.project.github && props.project.github.length > 0) {
        gitHubProjectLink = <IconButton aria-label="GitHub" target={'_blank'} href={props.project.github}>
            <Avatar aria-label="Project" src={'/assets/projects/github-logo.png'}
                className={classes.smallAvatar}/>
        </IconButton>;
    }

    let companyName = null;
    if (props.project.companyName && props.project.companyName.length > 0) {
        companyName = <Box width="90%" display="flex" flexDirection="row" justifyContent="flex-end">
            <div>Company: {props.project.companyName}</div>
        </Box>;
    }

    let website = null;
    if (props.project.website && props.project.website.length > 0) {
        website = <IconButton aria-label="Website" target={'_blank'} href={props.project.website}>
            <PublicIcon/>
        </IconButton>;
    }

    let longDescription = null;
    if (props.project.longDescription && props.project.longDescription.length > 0) {
        longDescription = <Grid container justify={'flex-end'}>
            <ScrollDialog title={props.project.name}>
                <DetailedProject description={props.project.longDescription}/>
            </ScrollDialog>
        </Grid>;
    }
    return (
        <Card className={classes.card}>
            <CardHeader
                avatar={avatarProject}
                title={props.project.name}
                subheader={dateCreated}
                content={'test'}
            />
            {companyName}
            {slider || cardImage}
            <CardContent>
                <Typography variant="body2" color="textSecondary" component="p">
                    {props.project.shortDescription}
                </Typography>
            </CardContent>
            <CardActions disableSpacing>
                {gitHubProjectLink}
                {website}
                {longDescription}
            </CardActions>
        </Card>
    );
}
