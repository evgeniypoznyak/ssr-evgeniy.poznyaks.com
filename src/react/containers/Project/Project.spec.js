import React from 'react';
import {mount, shallow} from 'enzyme';
import {skills} from '../../shared/api';
import {Grid} from '@material-ui/core';
import Project from '../Project/Project';
import ScrollDialog from '../../components/UI/Dialog/ScrollDialog';
import DetailedProject from './DetailedProject/DetailedProject';

let wrapper;
let mountWrapper;

const project = skills.slice()[0].projects[0];

beforeAll(() => {
    wrapper = shallow(<Project project={project}/>);
    mountWrapper = mount(<Project project={project}/>);
});

describe('<Project/>', () => {
    it('renders correctly', () => {
        expect(wrapper.children.length).toBe(1);
        expect(mountWrapper.find(Grid)).toHaveLength(1);
        expect(wrapper.getElement()).toMatchSnapshot();
    });

    it('renders ScrollDialog', () => {
        const scrollDialog = mountWrapper.find(ScrollDialog);
        expect(mountWrapper.text()).toMatch('.');
        expect(scrollDialog).toHaveLength(1);
    });

    it('renders DetailedProject', () => {
        const detailedProject = wrapper.find(DetailedProject);
        expect(detailedProject).toHaveLength(1);
    });
});
