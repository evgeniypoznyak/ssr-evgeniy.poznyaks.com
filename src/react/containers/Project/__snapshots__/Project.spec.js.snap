// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Project/> renders ScrollDialog 1`] = `ReactWrapper {}`;

exports[`<Project/> renders correctly 1`] = `
<ForwardRef(WithStyles)
  className="makeStyles-card-2"
>
  <ForwardRef(WithStyles)
    avatar={
      <ForwardRef(WithStyles)
        aria-label="Project"
        className="makeStyles-avatar-6"
        src="https://media.keyshot.com/uploads/2018/10/keyshot-icon-256.png"
      />
    }
    subheader="May 29, 2013"
    title="E and K Photography"
  />
  <div
    className="makeStyles-sliderRoot-1"
  >
    <ForwardRef(WithStyles)
      className="makeStyles-header-8"
      elevation={0}
      square={true}
    >
      <ForwardRef(WithStyles)
        variant="caption"
      >
        Main page
      </ForwardRef(WithStyles)>
    </ForwardRef(WithStyles)>
    <img
      alt="Main page"
      className="makeStyles-img-9"
      src="https://images.unsplash.com/photo-1537944434965-cf4679d1a598?auto=format&fit=crop&w=400&h=250&q=60"
    />
    <ForwardRef(WithStyles)
      activeStep={0}
      backButton={
        <ForwardRef(WithStyles)
          disabled={true}
          onClick={[Function]}
          size="small"
        >
          <Memo />
          Back
        </ForwardRef(WithStyles)>
      }
      nextButton={
        <ForwardRef(WithStyles)
          disabled={false}
          onClick={[Function]}
          size="small"
        >
          Next
          <Memo />
        </ForwardRef(WithStyles)>
      }
      position="static"
      steps={5}
      variant="text"
    />
  </div>
  <ForwardRef(WithStyles)>
    <ForwardRef(WithStyles)
      color="textSecondary"
      component="p"
      variant="body2"
    >
      In very beginning I created this website from scratch. It wasn't like it look like today, but back in the days my first sandbox live was this website.When I started write script, I didn't choice any library or framework, it was in pure php. Today this website is completely different.
    </ForwardRef(WithStyles)>
  </ForwardRef(WithStyles)>
  <ForwardRef(WithStyles)
    disableSpacing={true}
  >
    <ForwardRef(WithStyles)
      aria-label="GitHub"
      href="https://github.com/evgeniypoznyak/eandkphoto.com"
      target="_blank"
    >
      <ForwardRef(WithStyles)
        aria-label="Project"
        className="makeStyles-smallAvatar-7"
        src="https://pngimg.com/uploads/github/github_PNG42.png"
      />
    </ForwardRef(WithStyles)>
    <ForwardRef(WithStyles)
      aria-label="Website"
      href="https://eandkphoto.com"
      target="_blank"
    >
      <Memo />
    </ForwardRef(WithStyles)>
    <ForwardRef(WithStyles)
      container={true}
      justify="flex-end"
    >
      <ScrollDialog
        title="E and K Photography"
      >
        <DetailedProject
          description={
            Array [
              Object {
                "description": "In very beginning I created PHP website from scratch. I didn't choice any library or framework, it was in pure php. In very beginning I created PHP website from scratch. I didn't choice any library or framework, it was in pure php.",
                "picture": "https://hygger.io/blog/wp-content/uploads/2018/05/Scrum-3.jpg",
              },
              Object {
                "description": "In very beginning I created PHP website from scratch. I didn't choice any library or framework, it was in pure php. In very beginning I created PHP website from scratch. I didn't choice any library or framework, it was in pure php.",
                "picture": "https://hygger.io/blog/wp-content/uploads/2018/05/Scrum-3.jpg",
              },
            ]
          }
        />
      </ScrollDialog>
    </ForwardRef(WithStyles)>
  </ForwardRef(WithStyles)>
</ForwardRef(WithStyles)>
`;
